src/pyshark/capture/live_capture.py

```diff
@@ -51,8 +51,12 @@ def get_parameters(self, packet_count=None):
         params = super(LiveCapture, self).get_parameters(packet_count=packet_count)
         if self.bpf_filter:
             params += ['-f', self.bpf_filter]
-        for interface in self.interfaces:
-            params += ['-i', interface]
+        if self.monitor_mode:
+            params += ['-I', '']
```

Comment:
Is there a reason for the else on line 60?

If means when in monitor_mode it'll default to eth0 and not the interface specified.

```
        if self.bpf_filter:
            params += ['-f', self.bpf_filter]
        if self.monitor_mode:
            params += ['-I']

        for interface in self.interfaces:
                params += ['-i', interface]

        return params
```

The above works well for me.